server/index.js

```diff
@@ -68,6 +68,86 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  const sql = `
+    select *
+    from "carts"`;
```

Comment:
In this endpoint, you need to first check if there is an existing `cartId` in the `req.session`. If there is not, then just send an empty array. otherwise, use the query provided in the instructions to get the cart items.